Docs: update mysql docs to reflect editor changes (#54568)
grafana/grafana: docs/sources/datasources/mysql.md

You can use wildcards (`*`) in place of database or table if you want to grant access to more databases and tables.

> ## Query Builder

Reviewer: Suggesting sentence case heading. 
```suggestion
## Query builder
```

{{< figure src="/static/img/docs/v92/mysql_query_builder.png" class="docs-image--no-shadow" }}

> The MySQL query builder is available when editing a panel using a MySQL datasource. The built query can be run by pressing the `Run query` button in the top right corner of the editor.

Reviewer: ```suggestion
The MySQL query builder is available when editing a panel using a MySQL data source. The built query can be run by pressing the `Run query` button in the top right corner of the editor.
```

### Format

> The response from MySQL can be formated as either a table or as a time series. To use the time series format one of the columns must be named `time`.

Reviewer 2: ```suggestion
The response from MySQL can be formatted as either a table or as a time series. To use the time series format one of the columns must be named `time`.
```

### Dataset and Table selection

> In the dataset dropdown you can chose the MySQL database to query. The dropdown will be populated with the databases that the user has access to.
> When the dataset is selected the table dropdown is populated with the tables that are available.

Reviewer 2: ```suggestion
In the dataset dropdown, choose the MySQL database to query. The dropdown is be populated with the databases that the user has access to.
When the dataset is selected, the table dropdown is populated with the tables that are available.
```

### Columns and Aggregation functions (SELECT)

> Using the dropdown you can select a column to include inte the data. You can also specify a an optional aggregation function.

Reviewer 2: ```suggestion
Using the dropdown,  select a column to include in the data. You can also specify an optional aggregation function.
```

> You may add further value columns by clicking the plus button and another column dropdown will appear.

Reviewer 2: ```suggestion
Add further value columns by clicking the plus button and another column dropdown appears.
```

### Filter data (WHERE)

> To add a filter flip the switch at the top of the editor.
> Using the first dropdown you can select if all the filters need to match (AND) or if only one of the filters need to match (OR).
> 
> To add more columns to filter on use the plus button.

Reviewer 2: ```suggestion
To add a filter, flip the switch at the top of the editor.
Using the first dropdown, select if all the filters need to match (AND) or if only one of the filters needs to match (OR).

To add more columns to filter on use the plus button.
```

### Group By

> To group the results by a column, flip the group switch at the top of the editor. You can then choose what column to group the results by. The group by clause can be removed by pressing the X button
> 

Reviewer 2: ```suggestion
To group the results by column, flip the group switch at the top of the editor. You can then choose which column to group the results by. The group by clause can be removed by pressing the X button.

```

### Preview

> By flipping the preview switch at the top of the editor you can get a preview of the SQL query generated by the query builder.

Reviewer 2: ```suggestion
By flipping the preview switch at the top of the editor, you can get a preview of the SQL query generated by the query builder.
```

## Code editor

{{< figure src="/static/img/docs/v92/mysql_code_editor.png" class="docs-image--no-shadow" }}

> To make advanced queries you can switch to the code editor by clicking `code` in the top right corner of the editor. The code editor support autocompletion of tables, columns, SQL keywords, standard sql functions, grafana template variables and grafana macros. Columns cannot be completed before a table has been specified.

Reviewer 2: ```suggestion
To make advanced queries, switch to the code editor by clicking `code` in the top right corner of the editor. The code editor support autocompletion of tables, columns, SQL keywords, standard sql functions, Grafana template variables and Grafana macros. Columns cannot be completed before a table has been specified.
```
